styles/google-classroom/catppuccin.user.css

```diff
@@ -0,0 +1,1100 @@
+    .LXctle.VfPpkd-scr2fc-OWXEXe-uqeOfd:enabled .VfPpkd-uMhiad::after {
+      background: @text!important;
+    }
+
+    .LXctle.VfPpkd-scr2fc-OWXEXe-uqeOfd:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe, :active)
+      .VfPpkd-uMhiad::after {
+      background: @overlay2!important;
+    }
+
+    .LXctle:enabled:hover:not(.VfPpkd-ksKsZd-mWPk3d-OWXEXe-AHe6Kc-XpnDCe, :active)
+      .VfPpkd-l6JLsf::before {
+      background: @overlay0!important;
+    }
+
+    /* classroom title on hover */
+    .CbuVcc:focus .Pce5Kb,
+    .CbuVcc:hover .Pce5Kb {
+      color: @green;
+    }
+
+    .UvHKof .uO32ac,
+    .UvHKof .ypv4re,
+    .Iwp0Ue:not(:first-child),
+    .u73Apc {
+      border-color: @surface1;
+    }
+
+    /* Loading bar */
+    nav {
+      .bFjUmb-Wvd9Cc,
+      .CNpREd.bFjUmb-Wvd9Cc {
+        background-color: @accent-color;
+      }
+
+      .g2MItd.bFjUmb-Wvd9Cc,
+      .g2MItd.CNpREd.bFjUmb-Wvd9Cc {
+        background-color: @peach;
+      }
+
+      .UvHKof.bFjUmb-Wvd9Cc,
+      .UvHKof.CNpREd.bFjUmb-Wvd9Cc {
+        background-color: @surface1;
+      }
+
+      .g2MItd.bFjUmb-Wvd9Cc,
+      .g2MItd.CNpREd .bFjUmb-Wvd9Cc {
+        background-color: @peach;
+      }
+    }
+
+    .bFjUmb-Ysl7Fe,
+    .CNpREd .bFjUmb-Ysl7Fe {
+      background-color: @surface0;
+    }
+
+    .g2MItd.VnOHwf-Tvm9db,
+    .g2MItd .VnOHwf-Tvm9db,
+    .g2MItd.CNpREd.VnOHwf-Tvm9db,
+    .g2MItd.CNpREd .VnOHwf-Tvm9db,
+    .g2MItd .wwnMtb:not(.RDPZE),
+    .g2MItd .OZ6W0d:not(.RDPZE),
+    .g2MItd .FL3Khc:not(:disabled),
+    .g2MItd .n42Gr:not(:disabled),
+    .g2MItd .OGhwGf:hover,
+    .g2MItd .DqwBN:not(.RDPZE),
+    .g2MItd .YhQJj:not(.RDPZE),
+    .g2MItd .K2V86d:not(.RDPZE),
+    .g2MItd .l3F1ye:not(.RDPZE),
+    .g2MItd .An19kf:not(.RDPZE),
+    .g2MItd .BEAGS:not(.RDPZE),
+    .g2MItd .AeAAkf:not(.RDPZE) {
+      color: @peach;
+      fill: @peach;
+    }
+
+    .g2MItd.bFjUmb-Wvd9Cc.Fh209b:not(li),
+    .g2MItd .bFjUmb-Wvd9Cc.Fh209b:not(li),
+    .g2MItd .bDxw8b:not(:disabled),
+    .g2MItd.CNpREd.Fh209b .bFjUmb-Tvm9db:not(li .bFjUmb-Tvm9db) {
+      background-color: @peach;
+    }
+
+    .g2MItd.bFjUmb-Ysl7Fe.Fh209b,
+    .g2MItd .bFjUmb-Ysl7Fe.Fh209b,
+    .g2MItd .Iwp0Ue:not(.xp2dJ, .rZXyy):hover.rXdid,
+    .g2MItd .Iwp0Ue:not(.xp2dJ, .rZXyy):hover.h7Ww0,
+    .g2MItd .Iwp0Ue:not(.xp2dJ, .rZXyy):hover .rXdid,
+    .g2MItd .Iwp0Ue:not(.xp2dJ, .rZXyy):hover .h7Ww0,
+    .g2MItd .xWw7yd:hover.rXdid,
+    .g2MItd .xWw7yd:hover.h7Ww0,
+    .g2MItd .xWw7yd:hover .rXdid,
+    .g2MItd .xWw7yd:hover .h7Ww0,
+    .g2MItd .Iwp0Ue:not(.xp2dJ):focus-within.JiTjnd,
+    .g2MItd .Iwp0Ue:not(.xp2dJ):focus-within.h7Ww0,
+    .g2MItd .Iwp0Ue:not(.xp2dJ) :focus-within.JiTjnd,
+    .g2MItd .Iwp0Ue:not(.xp2dJ) :focus-within.h7Ww0,
+    .g2MItd .xWw7yd:focus-within.JiTjnd,
+    .g2MItd .xWw7yd:focus-within.h7Ww0,
+    .g2MItd .xWw7yd :focus-within.JiTjnd,
+    .g2MItd .xWw7yd :focus-within.h7Ww0 {
+      background-color: mix(@peach, @base, 15%);
```

Comment:
```suggestion
      background-color: mix(@peach, @yellow);
```
That usually does the trick for orange.

Reply:
I mixed base in to make a darker color for backgrounds, such as here:
![image](https://github.com/catppuccin/userstyles/assets/87151697/2cb3de90-1591-4ea2-a36a-d07b577dc3f1)

Reply:
Are we talking about the orange in the circle with the G or the backround?

Reply:
@BomberFish

Reply:
The background, the circle just uses regular peach

Reply:
@uncenter

Reply:
Why can we not use `fade` or `darken` here?

Reply:
@uncenter I just made some changes, let me know what you think.

Reply:
If it looks the same / similar to you then that is fine.